### ISSUE TITLE
Free Text parser ,wont work in certain cases as expected

### ISSUE THREAD

**Issue by FurqanKhan1** [MISP/MISP#3011]
Hi @iglocska @adulau 

While just ingesting / integrating a new feed source i discovered that the  free text parser seems to parse an ioc incorrectly if the ioc would not be properly delemeted with spaces .

For example in the following case if i give the input like as under 

2016-08-01 13:06:05	sip=177.139.250.177	dport=23

The parser tags /identifies the type as file-name and parses it as 

**sip=177.139.250.177**	

The feed source is http://data.netlab.360.com/feeds/mirai-scanner/scanner.list ,
I have tried to put it as a csv feed also and specified the delemeter as "=" ,in that case the parser produces an out put like **109.86.182.249	dport** .

Any thoughts as to how should i go about it.

Thanks for your help as always !

**Comment by adulau** [comment]
Good point. We might need to update the regexp part of the freetext import.

In the mean time, you can use the CSV import and split on "=" and it should work as expected then.

Thanks for the link, I'll add this feed in the default feeds.

**Comment by FurqanKhan1** [comment]
Hi @adulau ,thanks for the prompt response .
I did try with the csv option ,where i delemeted over "=" ,but its taking the dport string as well 
**"109.86.182.249 dport"**  ,is what it would store it as if we specify the delimiter / split as = ,...

Thanks !

**Comment by iglocska** [comment]
yeah "=" is not used as a delimiter in the text parser. The only danger I see with that is URLs containing "=" - do we want to split on that?

**Comment by FurqanKhan1** [comment]
Can we establish something like checking if the ioc starts with the URL scheme <http/https> or matches a url pattern ,after we split on the newlines and space delemeters (where we just have an actual word string to process further)then we ignore splitting on = ,otherwise we do it..

Would it break anything !

**Comment by adulau** [comment]
We did some test. Indeed having two delimiters would help in this case. @iglocska has some ideas ;-)

**Comment by SteveClement** [comment]
A good freetext import test is plainly copy/pasting the MISP Install instructions txt file into it.

It will go crazy on some attributes ;)
